ipatests/test_integration/test_installation.py

```diff
@@ -444,6 +457,64 @@ def test_ipactl_restart_pki_tomcat(self):
         pki_pid_after_restart_2 = get_pki_tomcatd_pid(self.master)
         assert pki_pid_after_restart != pki_pid_after_restart_2
 
+    def test_ipactl_scenario_check(self):
+        """ Test if ipactl starts services stopped by systemctl
+
+        This will first check if all services are running then it will stop
+        few service. After that it will restart all services and then check
+        the status and pid of services.It will also compare pid after ipactl
+        start and restart in case of start it will remain unchanged on the
+        other hand in case of restart it will change.
+        """
+        # listing all services
+        services = ["Directory", "krb5kdc", "kadmin", "named", "httpd",
+                    "ipa-custodia", "pki-tomcatd", "ipa-otpd",
+                    "ipa-dnskeysyncd"]
+
+        # checking the service status
+        cmd = self.master.run_command(['ipactl', 'status'])
+        for service in services:
+            assert "{} Service: RUNNING".format(service) in cmd.stdout_text
+
+        # stopping few services
+        service_stop = ["krb5kdc", "kadmin", "httpd"]
+        for service in service_stop:
+            self.master.run_command(['systemctl', 'stop',
+                                           '{}'.format(service)])
+
+        # checking service status
+        service_start = ["Directory", "ipa-custodia", "pki-tomcatd",
+                         "ipa-otpd", "ipa-dnskeysyncd", "named"]
+        cmd = self.master.run_command(['ipactl', 'status'])
+        for service in service_start:
+            assert "{} Service: RUNNING".format(service) in cmd.stdout_text
+        for service in service_stop:
+            assert "{} Service: STOPPED".format(service) in cmd.stdout_text
+
+        # starting all services again
+        self.master.run_command(['ipactl', 'start'])
+
+        # get process id of services
+        ipa_services_pid = get_ipa_services_pid(self.master)
+
+        # restarting all services again
+        self.master.run_command(['ipactl', 'restart'])
+
+        # check if pid for services are different
+        services_pid_after_restart = get_ipa_services_pid(self.master)
+        assert ipa_services_pid != services_pid_after_restart
+
+        # checking service status
+        cmd = self.master.run_command(['ipactl', 'status'])
+        for service in services:
+            assert "{} Service: RUNNING".format(service) in cmd.stdout_text
+
+        # starting all services again
+        self.master.run_command(['ipactl', 'start'])
+
+        # check if pid for services are same
+        assert services_pid_after_restart == get_ipa_services_pid(self.master)
```

Comment:
Shouldn't it be `services_pid_after_start` instead of `services_pid_after_restart`?